velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -50,7 +79,8 @@ class SubstraitVeloxPlanConverter {
       u_int32_t& index,
       std::vector<std::string>& paths,
```

Comment:
Consider changing this API to take a reference to SplitInfo struct to return a pair of PlanNodePtr and a SplitInfo